### ISSUE TITLE
Cannot Change Max Players

### ISSUE THREAD

**Issue by HaasCM** [HaasCM/quebec_one#12]
We cannot change the max players of the server and mission file

**Comment by HaasCM** [comment]
This has something to do with sqm file compatibility.

**Comment by SirBaconPoop** [comment]
Yeah I've noticed this as well, there is a parameter in the eden editor that is preventing this as well as the description code. I dont think this is a problem since the server.cfg already defines 32 players as max
